core/src/main/java/com/scalar/db/transaction/consensuscommit/CommitHandler.java

```diff
@@ -350,7 +372,7 @@ public void commitRecords(Snapshot snapshot) {
       }
       parallelExecutor.commitRecords(tasks, snapshot.getId());
     } catch (Exception e) {
-      logger.warn("Committing records failed. Transaction ID: {}", snapshot.getId(), e);
+      logger.info("Committing records failed. Transaction ID: {}", snapshot.getId(), e);
```

Comment:
Changed log level from `warn` to `info` because it is a normal occurrence and failing to commit records does not affect correctness.